docs/user-event/intro.mdx

```diff
@@ -68,7 +68,7 @@ test('trigger some awesome feature when clicking the button', async () => {
   // See https://testing-library.com/docs/dom-testing-library/install#wrappers
   render(<MyComponent />)
 
-  await user.click(screen.getByRole('button', { name: /click me!/i }))
+  await userEvent.click(screen.getByRole('button', { name: /click me!/i }))
```

Comment:
`user` is correct here, for more info see https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent